okonomiyaki/platforms/__init__.py

```diff
@@ -1,9 +1,11 @@
 # flake8: noqa
 from .abi import PlatformABI, default_abi
 from .epd_platform import X86, X86_64, EPDPlatform, applies
-from .platform import Platform, OSKind, FamilyKind, NameKind
+from ._platform import Platform, OSKind, FamilyKind, NameKind
```

Comment:
> okonomiyaki.platforms/okonomiyaki.file_formats/okonomiyaki.versions

if that was the case then why is `epd_platform` public and `_platform` not? It looks inconsistent.